openedx_events/apps.py

```diff
@@ -14,14 +14,20 @@ def general_signal_handler(sender, signal, **kwargs):  # pylint: disable=unused-
     """
     Signal handler for publishing events to configured event bus.
     """
-    configurations = getattr(settings, "EVENT_BUS_PRODUCER_CONFIG", {}).get(signal.event_type, ())
+    event_type_publish_configs = getattr(settings, "EVENT_BUS_PRODUCER_CONFIG", {}).get(signal.event_type, {})
```

Comment:
Using `publish` in the variable name and `PRODUCER` in the setting name is an unnecessary inconsistency.